test/visual/index.html

```diff
@@ -247,10 +247,7 @@ <h1>Visual Tests</h1>
       function localRendering (name, callback) {
         var canvas = create('canvas', { width: opt.width, height: opt.height, title: name })
         var tests = window.tests
-        var ctx = canvas.getContext('2d', {
-          alpha: true,
-          willReadFrequently: typeof opt.gpu == 'boolean' ? !opt.gpu : undefined
```

Comment:
This was deliberate as the `willReadFrequently=true` flag will disable GPU rendering on browser (assuming it was ever enabled in settings). As you said, things can be drawn slightly different with or w/out GPU.  The parallel vertical lines test is one of those, for example.